Realm/RLMObject.h

```diff
@@ -203,12 +203,12 @@ RLM_ASSUME_NONNULL_BEGIN
 /**
  The Realm in which this object is persisted. Returns nil for standalone objects.
  */
-@property (nonatomic, readonly, nullable) RLMRealm *realm;
+@property (nonatomic, readonly, nullable, getter = realm) RLMRealm *realm;
```

Comment:
That's an oversight on my part. I saw the explicit getters for `invalidated` and `deletedFromRealm` and for some reason thought at the time I needed to add it (but the ivar isn't named differently...). What is missing is that I wanted to add `realm` and `objectSchema` properties to `RLMObjectBase` since that was the point of moving the getters to there. I will correct.